app/assets/javascripts/backbone/views/calendar_view.js.coffee

```diff
@@ -4,27 +4,39 @@ class MS.Views.CalendarView extends Backbone.View
 
   initialize: (params) ->
     # Initialize calendar
-
     @$calendar = @$('#calendar')
 
     @$calendar.fullCalendar
       # Changes the default event render to load in html rather than title only
-      eventRender: @eventRender.bind(this),
-      loading: @loading.bind(this),
+      eventRender: @eventRender.bind(this)
+      loading: @loading.bind(this)
       events: params.calendar_events_url
       customButtons:
         legend:
           text: 'Legend'
       header:
         left: 'prev,next today'
         center: 'title'
-        right: 'month,agendaWeek legend'
+        right: 'month,basicWeek legend'
       allDayDefault: true
+      dayClick: @dayClick.bind(this)
 
     @renderLegend()
 
-  refresh: (e) ->
-    @$calendar.fullCalendar('refetchEvents')
+  events:
+    'click .loan-calendar .cal-step': 'showStepModal'
+
+  dayClick: ->
+    if (@$el.find('.loan-calendar'))
+      loanId = @$el.find('.loan-calendar').attr('data-loan-id').replace(/loan-/, '')
```

Comment:
Instead of `attr('data-loan-id')` you can do `data('loan-id')`. Always preferable to do that because if you set data attrib with `data()` it is only retrieveable with `data()`. `attr()` won't work.